nipype/pipeline/engine/workflows.py

```diff
@@ -425,6 +425,9 @@ def write_graph(self, dotfilename='graph.dot', graph2use='hierarchical',
                 base_dir = os.getcwd()
         base_dir = make_output_dir(base_dir)
         if graph2use in ['hierarchical', 'colored']:
+            if self.name[:1].isdigit(): # these graphs break if int
+                raise ValueError('{} graph failed, workflow name cannot begin '
+                                 'with a number'.format(graph2use))
```

Comment:
if workflow name cannot begin with a number, shouldn't this be a check when a workflow/node is created/cloned? why is a workflow name check in graph generation?